osu.Game/Skinning/LegacySkinExtensions.cs

```diff
@@ -82,8 +85,18 @@ protected override void LoadComplete()
                 if (timeReference != null)
                 {
                     Clock = timeReference.Clock;
-                    PlaybackPosition = timeReference.Clock.CurrentTime - timeReference.AnimationStartTime;
+                    ((IBindable<double>)animationStartTime).BindTo(timeReference.AnimationStartTime);
```

Comment:
Might be best to expose as a `Bindable` instead (in the interface) if this is meant to be a mutable bindable.

Reply:
The intention was to allow mutability in whoever is controlling the animation and disallow mutability in the animation itself by scoping to the interface, but I guess I can go either way.

Reply:
I guess it's neither here nor there. `Bindable`/`IBindable` still remaining a strange pattern in all usages